02.create-and-explore/06.all-about-scripting/03.avatar-scripts/docs.md

```diff
@@ -28,9 +28,7 @@ In order to attach a script to your avatar, you must modify your avatar's FST fi
 When packaging your model, the tool will iterate through the directory and add each script to the FST file. Now, you can upload the folder structure to the Marketplace or to your preffered hosting service.
 ## How to Stop an Avatar Script
  You can temporarily disable or restart your avatar script in the script interface (crtl + J) by pressing the close or reload button. 
- ![](\endFlow.png)
+ ![](endFlow.png)
  However, there is no way to permanently prevent the script from running, other then removing the script field from the FST file.
 ## Example Script
 Here is an example of an avatar that is rigged appropriately to use with a Flow script: [Mannequin with Hair](https://github.com/highfidelity/hifi-docs-grav-content/tree/master/02.create-and-explore/05.avatars/08.%20create-avatar-with-flow/mannequinHairTest8.fst).
```

Comment:
This isn't actually the right example -- since its an FST file. The more I think about it, the more I don't think we actually need any example (since we have the flow tutorial now). Let's just remove this section.